docs/general/administration/hardware-selection.md

```diff
@@ -14,16 +14,16 @@ For a Jellyfin server, we recommend the following:
 ### Normal Server From Scratch
 
 - CPU: 4 Threads
-    - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
-    - Please Avoid Rapsberry PIs and other low powered SBCs.
-    - Please Avoid Intel Core M / Y series CPUs.
+  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
+  - Please Avoid Rapsberry PIs and other low powered SBCs.
+  - Please Avoid Intel Core M / Y series CPUs.
 - RAM: 8GB or more
 - Storage: SSD for Jellyfin files and transcoding cache.
 - Graphics: Intel > Nvidia > AMD > Apple.
-    - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
-    - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
-    - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.
-    - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.
+  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
+  - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
+  - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.
+  - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.
```

Comment:
As above: "is recommended" over "we recommend" here is better. It might also be worth thinking of framing it in terms of WHY it might be recommended, instead of just saying that X or Y is recommended. For example: "When using Intel based Mac hardware, installing Windows or Linux enables features that Mac OS does not, and will better support transcoding and hardware acceleration."

Reply:
This section is meant to be a TL;DR so I think the explanation should live somewhere else.